ccd-definition/ComplexTypes/RepaymentPlanLRspec-nonprod.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "ID": "RepaymentPlanLRspec",
+    "ListElementCode": "repaymentPlanLabel",
```

Comment:
same comment as above

Reply:
Having a discussion with both and the change with be in base file as it is not impact the functionality. This file will be removed